<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1780</width>
                <height>1300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralWidget">
            <widget class="QPushButton" name="pushButton_open_webcam">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>60</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Open Webcam</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_close_webcam">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>160</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Close Webcam</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_open_file">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>290</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Select second media</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_poly">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>380</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Show poly</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_test">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>465</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Test mode</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_swap">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>550</y>
                        <width>171</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Swap mouth</string>
                </property>
            </widget>
            <widget class="QPushButton" name="pushButton_memorize">
                <property name="geometry">
                    <rect>
                        <x>90</x>
                        <y>635</y>
                        <width>170</width>
                        <height>81</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Screen neutral face</string>
                </property>
            </widget>
            <widget class="QLabel" name="label">
                <property name="geometry">
                    <rect>
                        <x>310</x>
                        <y>60</y>
                        <width>640</width>
                        <height>480</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label2">
                <property name="geometry">
                    <rect>
                        <x>1059</x>
                        <y>60</y>
                        <width>731</width>
                        <height>431</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label3">
                <property name="geometry">
                    <rect>
                        <x>1059</x>
                        <y>540</y>
                        <width>731</width>
                        <height>431</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
            <widget class="QLabel" name="label4">
                <property name="geometry">
                    <rect>
                        <x>310</x>
                        <y>550</y>
                        <width>731</width>
                        <height>431</height>
                    </rect>
                </property>
                <property name="text">
                    <string/>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>862</y>
                    <width>1110</width>
                    <height>18</height>
                </rect>
            </property>
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources/>
    <connections/>
</ui>